01-syntax/09-modules/return-to-electronic-life/index.html

```diff
@@ -0,0 +1,14 @@
+<!doctype html>
+<head>
+    <script src = "elife.js"></script>
```

Comment:
The question is : How your app works in a browser ?
I don`t see imports of other modules ...